src/App.tsx

```diff
@@ -1,14 +1,45 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
+
 import { Loader } from './components/Loader';
+import { Todo, TodoFilterBy } from './types/Todo';
+import { getTodos } from './api';
+import { TodoModal } from './components/TodoModal';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoId, setTodoId] = useState(0);
+  const [query, setQuery] = useState('');
+  const [filter, setFilter] = useState(TodoFilterBy.NONE);
+  const [isLoading, setisLoading] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setisLoading] = useState(false);
```
isLoading is false until you start fetching data